ops/gauntlet/gauntlet_plus_plus_starknet.go

```diff
@@ -0,0 +1,313 @@
+package gauntlet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"context"
+	"github.com/rs/zerolog/log"
+	g "github.com/smartcontractkit/gauntlet-plus-plus/sdks/go-gauntlet/client"
+)
+
+var (
+	sgpp *StarknetGauntletPlusPlus
+)
+
+type Request struct {
+	Input map[string]any 		 `json:"input"`
+	Command string					 `json:"command"`
+}
+
+type StarknetGauntletPlusPlus struct {
+	client				*g.ClientWithResponses
+	gr      			*http.Response
+	providers 		*[]g.Provider
+}
+
+func (sgpp *StarknetGauntletPlusPlus) BuildProviders(address string, rpcUrl string, privateKey string) (*[]g.Provider) {
+	var input map[string]*interface{}
```

Comment:
people tend to use `any` or `interface{}` now

Reply:
main benefit is to make the code look more readable

Reply:
>people tend to use any or interface{} now

g.Provider requires input to be of type `map[string]interface{}`

Reply:
I'll clean it up and push a revision